### DISCUSSION TITLE
复制的节点粘贴后能否自动化布局？且不影响节点的展示。类似于思维导图新增一个子节点，会自动平分父节点。

### DISCUSSION THREAD

**Discussion by xiazhaohui** [antvis/X6#2070] [Q&A] [answered]
复制的节点粘贴后能否自动化布局？且不影响节点的展示。类似于思维导图新增一个子节点，会自动平分父节点。
复制一个B1节点：粘贴后的B2节点可以设置偏移量，但是不能和被B1节点一样，自动化布局到父节点的下级位置。

<img width="484" alt="iShot2022-04-28_11 25 28" src="https://user-images.githubusercontent.com/82207908/165670325-0da3e52f-3767-4db9-8223-6a7e271e18ac.png">

理想效果：B1和B2自动化布局到A节点的子级，类似思维导图效果
<img width="385" alt="iShot2022-04-28_11 27 25" src="https://user-images.githubusercontent.com/82207908/165670487-839c0bbb-81f8-4fed-9bed-ff64abd49f44.png">

**Comment by NewByVector** [comment, marked as answer]
现在需要手动执行布局方法，2.0 版本考虑将这个功能内置进来。

**Reply by xiazhaohui** [reply]
现阶段，如何去手动执行布局方法呢？有没有Demo啊，就是类似那种给一个节点新增或删除一个字节点后触发自动布局。
另外2.0版本大概啥时候上线呀
